src/lib/stdout-manipulator.ts

```diff
@@ -54,7 +54,23 @@ function color(line: string, noClear: boolean = false): string {
   return line;
 }
 
-export function print(noColors: boolean, noClear: boolean, line: string): void {
+type PrintParams = {
```

Comment:
please rename to `TPrintParams`

Reply:
@gilamran Done in 9f88210c206a9b2af5e4ec0e47dfa3543bd45555 (commit to squash)